plenum/recorder/src/simple_zstack_with_recorder.py

```diff
@@ -0,0 +1,47 @@
+import os
+from typing import Set
+
+from stp_core.common.log import getlogger
+
+from plenum.recorder.src.recorder import Recorder
+from storage.kv_store_leveldb_int_keys import KeyValueStorageLeveldbIntKeys
+from stp_zmq.simple_zstack import SimpleZStack
+
+logger = getlogger()
+
+
+class SimpleZStackWithRecorder(SimpleZStack):
+    # Used during recording
+    def __init__(self, *args, **kwargs):
+        parent_dir, _ = os.path.split(args[0]['basedirpath'])
+        name = args[0]['name']
+        from stp_core.network.keep_in_touch import KITNetworkInterface
+        if isinstance(self, KITNetworkInterface):
+            db_path = os.path.join(parent_dir, 'data', name, 'recorder')
+        else:
+            db_path = os.path.join(parent_dir, 'data', name[:-1], 'recorder')
```

Comment:
Why the path is different here?

Reply:
Nodestack and clientstack recorders go in different directories.